src/components/charts/heatMap3DPlot/heatMap3DScatterPlot.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Nouislider from 'nouislider-react';
+import debounce from 'lodash/debounce';
+import MultiAxisScatter from './MultiAxisScatter';
+import 'nouislider/distribute/nouislider.css';
+import './heat-map-3d-plot.module.scss';
+
+class MultiAxisLayout extends React.PureComponent {
```

Comment:
As near as I can tell this component isn't used.  If so please remove.

Reply:
You are correct. I had pulled it over thinking I would need to use this one.